### PR TITLE
fix: resolve several issues

### PR DESCRIPTION
**Pull request by Gorter-dev**
Closes
https://github.com/Iconica-Development/flutter_notification_center/issues/8
https://github.com/Iconica-Development/flutter_notification_center/issues/10
https://github.com/Iconica-Development/flutter_notification_center/issues/15
https://github.com/Iconica-Development/flutter_notification_center/issues/19

### CODE REVIEW

File: packages/flutter_notification_center_firebase/lib/src/services/firebase_notification_service.dart [hunk 1]
```diff
@@ -62,8 +72,17 @@ class FirebaseNotificationService
   @override
   Future<List<NotificationModel>> getActiveNotifications() async {
     try {
+      var userId = FirebaseAuth.instance.currentUser?.uid;
```
Comment (freekvandeven) [inline packages/flutter_notification_center_firebase/lib/src/services/firebase_notification_service.dart:75]: Can you add an option to give a specific firebase app or firebaseAuth to the service

**Review by freekvandeven — CHANGES_REQUESTED** [review]
the firebase instance should be in the constructor of the notifications service

**Review by mighttymike — APPROVED** [review]

**Review by freekvandeven — CHANGES_REQUESTED** [review]
Also remove the test folder in the root and the pubspec.yaml. You could add a melos.yaml from another component if you want to indicate that it is a monorepo

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Copyright (c) 2023 Iconica, All rights reserved.
```
Comment (freekvandeven) [inline LICENSE:1]: it is actually 2024 already @Gorter-dev

**Review by freekvandeven — APPROVED** [review]

**Pull request merged**